libs/api/repository/src/lib/gn4/gn4-repository.ts

```diff
@@ -25,22 +26,32 @@ import {
 } from '@geonetwork-ui/common/domain/model/search'
 import { catchError, map, tap } from 'rxjs/operators'
 import {
+  assertValidXml,
   findConverterForDocument,
   Gn4Converter,
   Gn4SearchResults,
   Iso19139Converter,
 } from '@geonetwork-ui/api/metadata-converter'
 import { CatalogRecord } from '@geonetwork-ui/common/domain/model/record'
-import { HttpErrorResponse } from '@angular/common/http'
+import {
+  HttpClient,
+  HttpErrorResponse,
+  HttpHeaders,
+} from '@angular/common/http'
 
 const TEMPORARY_ID_PREFIX = 'TEMP-ID-'
 
+export const LOCAL_STORAGE_RECORD_PREFIX = 'me-record-draft-'
```

Comment:
This prefix should actually never be changed (unless a retrocompatibility system is done) because otherwise all existing drafts in users sessions will get lost and never be cleaned up

Reply:
But we are still in developpment we have no user right ?

So what should I do ? 

Remove this diff ? 

Add a comment saying that : 
> This prefix should actually never be changed (unless a retrocompatibility system is done) because otherwise all existing drafts in users sessions will get lost and never be cleaned up

????

Reply:
1. restore the previous prefix (which was also more explicit than "me-record-draft")
2. add a comment to emphasize the fact that changing this prefix is a breaking change

thanks!